hypothesis-python/src/hypothesis/extra/numpy.py

```diff
@@ -488,8 +480,9 @@ def dtype_factory(kind, sizes, valid_sizes, endianness):
 
 
 @defines_dtype_strategy
-def unsigned_integer_dtypes(endianness="?", sizes=(8, 16, 32, 64)):
-    # type: (str, Sequence[int]) -> st.SearchStrategy[np.dtype]
+def unsigned_integer_dtypes(
+    endianness: str = "?", sizes: Sequence[int] = (8, 16, 32, 64)
```

Comment:
Wouldn't be better here and in similar cases with constant values to use [Literal](https://mypy.readthedocs.io/en/latest/literal_types.html) annotation which is provided by typing extensions?

Reply:
I'd prefer not to add a runtime dependency which is only useful for type-checkers.

I think this PR is big enough already, but I'd be perfectly happy with a follow-up along the lines of 

```python
try:
    from typing import Literal
except ImportError:
    # Python <=3.7 without the typing_extensions package
    Endianness = str
    Size = int
    NDims = int
else:
    Endianness = Literal["?", "=", "<", ">"]
    Size = Literal[8, 16, 32, 64]
    NDims = Literal[0, 1, 2, 3, 4, ..., 31, 32]
```

Reply:
Current implementation is fine, just suggested this to not forget about this idea. Good point, that it's better to make in separate PR. Proposed solution is good for me.